### DISCUSSION TITLE
Subscriptions Issues

### DISCUSSION THREAD

**Discussion by DzNab** [joemasilotti/railsdevs.com#900] [Q&A]
Hello Joe,

After numerous attempts and many hours spent, I'm pleased to say that the app deployment is finally nearing completion.

Currently, I'm conducting tests using Stripe's test mode, but I've encountered an issue that I'm hoping you can help me with. I successfully created a product with a full-time plan and inserted the key into the credentials. Subscribing to the product appears to work without a hitch.

However, when I attempt to contact a user, I receive a message indicating that there's a need to upgrade my subscription. Upon examining the database, I noticed that a record has been created in pay_customers, and pay_webhooks has been populated, but there's no corresponding entry in pay_subscriptions.

I've invested several hours trying to resolve this issue, but to no avail. Have you encountered a similar problem before?

As a side note, I haven't modified the placeholder credentials on iOS and RevenueCat. Could this be the root cause of the problem? Or is it acceptable to leave them unchanged?

Thank you in advance for your assistance.

**Comment by joemasilotti** [comment]
It sounds like you have everything set up correctly. The only thing I can think to double check is the credentials and subscriptions configuration file to make sure they match what you have in Stripe.
